card.io/src/main/java/io/card/payment/DataEntryActivity.java

```diff
@@ -48,6 +49,7 @@
      * PayPal REST Apis only handle max 20 chars postal code, so we'll do the same here.
      */
     private static final int MAX_POSTAL_CODE_LENGTH = 20;
+    private static final int MAX_NAME_ON_CARD_LENGTH = 50;
```

Comment:
I couldn't find a standard max value but I can update it to 175 if that's the braintree max